wireshark/protofields.lua

```diff
@@ -0,0 +1,379 @@
+-- This file is auto-generated. Any hand edits will be overwritten!
```

Comment:
I don't love checking in an autogenerated file, but I think it's correct here, since it makes installing the plugin as simple as "clone repo, add symlink".